src/pal/src/cruntime/printfcpp.cpp

```diff
@@ -310,6 +326,11 @@ BOOL Internal_ExtractFormatA(CPalThread *pthrCurrent, LPCSTR *Fmt, LPSTR Out, LP
         {
             *Prefix = PFF_PREFIX_LONG;
         }
+        else if (**Fmt == 'l')
```

Comment:
You'll get a build break for BIT64 not defined (and you wouldn't support %ll in that case if you didn't get a build break).